### DISCUSSION TITLE
Expand the range of eligible colors for Most dazzling ride colors award

### DISCUSSION THREAD

**Discussion by SpartanFrederic104** [OpenRCT2/OpenRCT2#20161] [Feature requests]
With the 22 new color options, the game's total number of selectable colors (excluding cheat-exclusive colors) has been raised from 32 to 54. However, the eligible colors for the Most dazzling ride color schemes park award still consist of only four colors: bright green, bright purple, bright pink, and light orange. (this is confirmed by looking at [Award.cpp](https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/management/Award.cpp))

I feel like the range of colors that are eligible for the Most dazzling colors award should be expanded a bit to include some of the new color options. Any thoughts on this?

**Comment by MASHzj** [comment]
I would like at least yellow, red and blue to be added.

**Comment by HtotheTML** [comment]
I think the arrangement of eligible colors for the award should be intuitive to pick up on from the color selection screen. Something like this to begin with, sticking to basic colors:
![Illustration2](https://github.com/OpenRCT2/OpenRCT2/assets/108596959/a16e0cb7-b005-4ad4-bf4e-38a0a462917b)
This would up the eligible colors from 4 to 14.

**Reply by MASHzj** [reply]
I don't really see all colours in the third and six row as bright, so not sure about those.

**Reply by HtotheTML** [reply]
It's a tough case, since some of the colors in those rows are labeled as "bright" or "light."

Excluding those would drop this example to 7 eligible colors, and I'm not sure that really solves the proposed issue.

**Comment by autosysops** [comment]
Having the color be fixed from the start sounds quite boring. Personally I would prefer it way more if at the start of the game at random some colors where picked and these where saved in the savegame data. maybe create a subset of colors to pick at random so it doesn't on accident only pick brown colors.
